calc/package.scala

```diff
@@ -9,32 +9,73 @@ import dsl.*
 import distcompiler.calc.tokens.*
 
 object tokens:
-  object LowPrecOp extends Token:
+  object Expression extends Token:
     override def showSource: Boolean = true
 
-  object HighPrecOp extends Token:
+  object AddOp extends Token:
     override def showSource: Boolean = true
 
-  object Expression extends Token:
+  object DivOp extends Token:
+    override def showSource: Boolean = true
+
+  object MulOp extends Token:
+    override def showSource: Boolean = true
+
+  object SubOp extends Token:
+    override def showSource: Boolean = true
+
+  object Add extends Token:
+    override def showSource: Boolean = true
+
+  object Sub extends Token:
+    override def showSource: Boolean = true
+
+  object Mul extends Token:
+    override def showSource: Boolean = true
+
+  object Div extends Token:
     override def showSource: Boolean = true
 
   object Number extends Token:
     override def showSource: Boolean = true
 
 val wellformed: Wellformed =
   Wellformed:
-    Node.Top ::= repeated(choice(tokens.Number, tokens.Expression, tokens.LowPrecOp, tokens.HighPrecOp))
+    Node.Top ::= repeated(choice(tokens.Number, tokens.AddOp, tokens.SubOp, tokens.MulOp, tokens.DivOp))
```

Comment:
But also, this is what the Reader will produce, not the final grammar (which will likely be much more specific). This should probably only be in the reader, and does not need to reflect the final AST necessarily.